account_invoice_search_by_reference/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
```

Comment:
Just a minor thing but IMO the copyright should refer to authors or contributors who add new features to the module. Contributors that just migrate should go in the contributors section in the readme. Thanks.